packages/web3/test/e2e/get_protocol_version.test.ts

```diff
@@ -33,7 +33,8 @@ describe(`${getSystemTestBackend()} tests - getProtocolVersion`, () => {
 		await closeOpenConnection(web3);
 	});
 
-	it('should get the protocol version for the connected node', async () => {
+	// doesn't exists with lodestar provider
```

Comment:
pls ask Infrastructure team can they enable this RPC call?

Reply:
done